libsplinter/Cargo.toml

```diff
@@ -117,11 +119,13 @@ circuit-read = []
 circuit-template = []
 proposal-read = []
 connection-manager = ["matrix"]
+network-peer-manager = ["connection-manager", "network-ref-map"]
```

Comment:
This should be put in the list alphabetically

Reply:
fixed